scripts/add-admin.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+principal=$(dfx identity get-principal)
```

Comment:
I didn't want a way to add / remove service controllers through a public API yet, but we could easily just make this method that (add_controller instead of add_admin).  When it's add_admin seems like it's implied that the type is `ServiceControllerKind::Admin`

Reply:
Sorry, I meant can take the principal as an argument (since doesn't always have to be current identity)